### PR TITLE
Add translated docs(1~3) for iOS-Debugging.md

### PR DESCRIPTION
**Pull request by alledy**
Add translated contents in Korean for iOS-Debugging.md

There are subheads(h3 heading) in the first subject(`Using the Debugger`) but I haven't numbered. 
If it needed I'll fix. Thanks.

### CODE REVIEW

**Review by JeffGuKang — APPROVED** [review]

File: docs/Debugging/iOS-Debugging.md [hunk 1]
```diff
@@ -1,3 +1,187 @@
...
+
+After you click the Run button in the workspace toolbar and your app builds successfully, Xcode runs your app and starts a debugging session. You can debug your app directly within the source editor with graphical tools such as data tips and Quick Look for the value of variables.
+
+워크스페이스의 툴바에서 실행 버튼을 누르면 당신의 앱은 성공적으로 빌드될 것입니다. Xcode는 당신의 앱을 실행시키고 디버깅 세션을 시작합니다. `data tips` 같은 그래픽 툴이나 [Quick Look](<https://developer.apple.com/library/archive/documentation/DeveloperTools/Conceptual/debugging_with_xcode/chapters/quick_look_data_types.html>)을 제공하는 소스 에디터에서 앱을 직접 디버깅할 수 있습니다. 
+
+The debug area and the debug navigator let you inspect the current state of your running app and control its execution.
+
+디버그 영역과 탐색기는 당신이 실행하고 있는 앱의 현재 상태를 검사하도록 하고 그것의 실행을 통제하도록 합니다. 
+
+![](https://developer.apple.com/library/archive/documentation/ToolsLanguages/Conceptual/Xcode_Overview/Art/XC_O_DebugFeatures_2x.png)
```
Comment (Duckuism) [inline docs/Debugging/iOS-Debugging.md]: You are using hyperlinks of developert.apple.com. In this case, If the paths of links are changed, all images will be disappeared in our markdown files. I have made the Image directory in docs/Debugging. You should add the image files in the Image directory and change the paths of image sources just like `./Images/<:image source name>` after download the image sources.

EX) ``

File: docs/Debugging/iOS-Debugging.md [hunk 2]
```diff
@@ -1,3 +1,187 @@
+## 1. Using the Debugger
+
+## 1. 디버거 사용하기
+
+After you click the Run button in the workspace toolbar and your app builds successfully, Xcode runs your app and starts a debugging session. You can debug your app directly within the source editor with graphical tools such as data tips and Quick Look for the value of variables.
+
+워크스페이스의 툴바에서 실행 버튼을 누르면 당신의 앱은 성공적으로 빌드될 것입니다. Xcode는 당신의 앱을 실행시키고 디버깅 세션을 시작합니다. `data tips` 같은 그래픽 툴이나 [Quick Look](<https://developer.apple.com/library/archive/documentation/DeveloperTools/Conceptual/debugging_with_xcode/chapters/quick_look_data_types.html>)을 제공하는 소스 에디터에서 앱을 직접 디버깅할 수 있습니다.
```
Comment (Duckuism) [inline docs/Debugging/iOS-Debugging.md]: As you can often see in translation, it would seem natural in context to delete `당신의` or `너의` or `네가`.

**Review by Duckuism — CHANGES_REQUESTED** [review]
Great job! 👍🏻 Please leave your answer to my reviews.

File: docs/Debugging/iOS-Debugging.md (hunk 1 above)
Comment (alledy) [reply docs/Debugging/iOS-Debugging.md]: I didn't notice that point. I'll change. Thanks!

**Comment by JeffGuKang** [comment]
Good job. But work with new branch always even update code in your repository.
Updating the master branch directly can be confusing.

**Comment by Duckuism** [comment]
Refer to this screenshot.
스크린샷 2019-09-18 15 54 30

**Pull request closed without merging**